shell/src/test/java/org/apache/accumulo/shell/format/DeleterFormatterTest.java

```diff
@@ -133,6 +141,7 @@ public void testNo() throws IOException {
     assertTrue(formatter.hasNext());
   }
 
+  // Need to properly fix this test.
```

Comment:
What's this comment about?

Reply:
Related to your other comment below in `DeleterFormatter`.